requirements-ontology.txt

```diff
@@ -0,0 +1,2 @@
+-r requirements.txt
```

Comment:
Since some may integrate another reasoning-based methods (eg: neural-network based), I'd say ontology should be an optional plugin to pycram core operations.
I actually added it in requirements.txt but then decided to move out as @tomsch420's suggestion.